nima/webclient/webclient/src/test/java/io/helidon/nima/webclient/http1/ClientRequestImplTest.java

```diff
@@ -0,0 +1,461 @@
+    }
+
+    // @Test
+    void testMaxStatusLineLengthFail() {
+        Http1Client client = WebClient.builder()
+                .maxStatusLineLength(1)
+                .build();
+        validateFailedResponse(client, new FakeHttp1ClientConnection(), "HTTP Response did not contain HTTP status line");
+    }
+
+    // @Test
+    void testMaxHeaderLineLengthSuccess() {
+        Http1Client client = WebClient.builder()
+                .maxStatusLineLength(20)
+                .build();
+        validateSuccessfulResponse(client, new FakeHttp1ClientConnection());
+    }
+
+    @Test
+    void testMediaContext() {
+        Http1Client client = WebClient.builder()
+                .mediaContext(new CustomizedMediaContext())
+                .build();
+        validateSuccessfulResponse(client, new FakeHttp1ClientConnection());
+    }
+
+    @Test
+    void testChunk() {
+        String[] requestEntityParts = {"First", "Second", "Third"};
+
+        Http1ClientRequest request = getHttp1ClientRequest(Http.Method.PUT, "/test");
+        request.connection(new FakeHttp1ClientConnection());
+        Http1ClientResponse response = getHttp1ClientResponseFromOutputStream(request, requestEntityParts);
+
+        validateChunkTransfer(response, true, NO_CONTENT_LENGTH, String.join("", requestEntityParts));
+    }
+
+    @Test
+    void testNoChunk() {
+        String[] requestEntityParts = {"First"};
+        long contentLength = requestEntityParts[0].length();
+
+        Http1ClientRequest request = getHttp1ClientRequest(Http.Method.PUT, "/test")
+                .header(Http.Header.CONTENT_LENGTH, String.valueOf(contentLength));
+        request.connection(new FakeHttp1ClientConnection());
+        Http1ClientResponse response = getHttp1ClientResponseFromOutputStream(request, requestEntityParts);
+
+        validateChunkTransfer(response, false, contentLength, requestEntityParts[0]);
+    }
+
+    @Test
+    void testForcedChunkNoContentLength() {
+        String[] requestEntityParts = {"First"};
+
+        Http1ClientRequest request = getHttp1ClientRequest(Http.Method.PUT, "/test");
+        request.connection(new FakeHttp1ClientConnection());
+        Http1ClientResponse response = getHttp1ClientResponseFromOutputStream(request, requestEntityParts);
+
+        validateChunkTransfer(response, true, NO_CONTENT_LENGTH, requestEntityParts[0]);
+    }
+
+    @Test
+    void testForcedChunkTransferEncodingChunked() {
+        String[] requestEntityParts = {"First"};
+
+        Http1ClientRequest request = getHttp1ClientRequest(Http.Method.PUT, "/test")
+                .header(Http.HeaderValues.TRANSFER_ENCODING_CHUNKED);
+        request.connection(new FakeHttp1ClientConnection());
+        Http1ClientResponse response = getHttp1ClientResponseFromOutputStream(request, requestEntityParts);
+
+        validateChunkTransfer(response, true, NO_CONTENT_LENGTH, requestEntityParts[0]);
+    }
+
+    // @Test
```

Comment:
Commented tests?

Reply:
As mentioned above I'm currently testing a failure that only happens in the pipeline but unable to reproduce locally. The code will still change in the coming days as I conduct the testing.

Reply:
Identified that the issue is coming from sending expect-100-continue to the server so have disabled that feature for now, until that issue is fixed from the server side. All the commented test are now reinstated.